react/features/filmstrip/middleware.js

```diff
@@ -31,6 +34,23 @@ MiddlewareRegistry.register(({ getState }) => next => action => {
             return result;
         }
         break;
+
+    case SET_FILMSTRIP_VISIBLE:
```

Comment:
Here I'm commenting on the commit "[RN] Unpin participant and set last N to 1 if the filmstrip is hidden" only:

When I see `visible`, I think of whether the `Component` is visible/invisible at the moment (or will be made visible/invisible shortly and for all intents and purposes of the app it should be considered visible/invisible). Indeed, the Web currently toggles the filmstrip's `visible` when we press the respective button in the UI. We also have a `visible` on the toolbox and it toggles when we move the mouse on the Web, for example, or when we tap on the Conference.

So I always thought of `enabled` here rather than `visible`. For example, when we do merge Web and mobile as much as possible and if we decide to have a filmstrip `visible` toggle in the UI, we'll be unpinning and setting last N based on the `visible`. But that's not what we want. If the filmstrip is simple toggled between visible and invisible, we still want the rest of the app to operate as expected. Only if the filmstrip is disabled, then we want to app to operate differently. The same goes for the toolbox.

I realize it's a small distinction at the time of this writing, I merely wanted to share my perspective that not separating `visible` and `enabled` now will make it a touch harder for us to distinguish behavior which we want on `visible` form behavior which we want on `enabled`.